src/cardboard-ui.js

```diff
@@ -137,6 +138,8 @@ CardboardUI.prototype.listen = function(optionsCallback, backCallback) {
  * Builds the UI mesh.
  */
 CardboardUI.prototype.onResize = function() {
+  if (!WebVRConfig.ENABLE_CARDBOARD_UI) return false;
```

Comment:
that'll [happen now](https://github.com/borismus/webvr-polyfill/pull/53/files#diff-1c90ff38a08209f9ebd4d05d1e43358eR48) with the default config object I added to `src/main`